ui/raidboss/data/02-arr/trial/titan-nm.txt

```diff
@@ -0,0 +1,107 @@
+### Titan Story / Normal
+# -ii 5C5 3CD -p 28B:203.0  28C:603.0
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+### Phase 1 100->85%
+0.0 "Start"
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.0 "--sync--" sync /:Titan:368:/ window 2,0
+5.0 "Rock Buster" sync /:Titan:281:/

```

Comment:
Is there a particular reason you didn't set up a [`tankCleave()`](https://github.com/quisquous/cactbot/blob/a5d5de00566ce83529c52b172112fbfe45051b5b/resources/responses.js#L197) timeline trigger based on these log lines?

Reply:
Hmm you are correct, they are far enough apart that would be possible.